applications/cheersland.md

```diff
@@ -200,6 +200,15 @@ Source codes will reside in: https://github.com/cheersland
 * **FTE:**  4
 * **Costs:** $3,000 equivalent
 
+**Goal - Implement Game module and logic in the Polkadot ecosystem.**  
+In order to implement whole Game logic for Polkadot & Kusama, and be more compatible with crypto assets such as DOT and KSM, we will deploy smart contracts on Plasm. The reason why we choose Plasm is that Plasm can provide faster transaction speed, lower trading fee and higher flexibility, which fits perfectly with our game.  

```

Comment:
Will your contracts be written in Solidity or `ink!`?
Also, just FYI, Plasm is now called [Astar](https://astar.network/).

Reply:
Thanks for your reply sir @mmagician , we will use Solidity coz it can be more convenient for us and save the time of development. For Aster, thanks for your kind reminder, we have push a new commit to resolve these comments.